src/main/scala/temple/generate/language/service/go/GoServiceGenerator.scala

```diff
@@ -2,22 +2,26 @@ package temple.generate.language.service.go
 
 import temple.generate.language.service.ServiceGenerator
 import temple.generate.language.service.adt._
+import temple.generate.utils.CodeTerm
 import temple.utils.FileUtils.{File, FileContent}
 import temple.utils.StringUtils.tabIndent
 
+/** Implementation of [[ServiceGenerator]] for generating Go */
 object GoServiceGenerator extends ServiceGenerator {
 
   private def generatePackage(packageName: String): String = s"package $packageName\n\n"
 
+  /** Given a service name, module name and whether the service uses inter-service communication, return the import block */
   private def generateImports(serviceName: String, module: String, comms: Boolean): String = {
     val sb = new StringBuilder
 
-    sb.append("""import (
-    |	"encoding/json"
-    |	"flag"
-    |	"fmt"
-    |	"log"
-    |	"net/http"""".stripMargin + "\n\n")
+    sb.append("import (" + "\n")
+    sb.append(tabIndent(""""encoding/json"""") + "\n")
+    sb.append(tabIndent(""""flag"""") + "\n")
+    sb.append(tabIndent(""""fmt"""") + "\n")
+    sb.append(tabIndent(""""log"""") + "\n")
+    sb.append(tabIndent(""""net/http"""") + "\n")
+    sb.append("\n")
```

Comment:
Now factor out all the repetition (including below) and I’ll be very happy

Reply:
Something like this? n.b. `spaced` will need to be adapted to support tabs, and named imports will need to be added (tuple with optional string?). This could be bundled up into a go-specific set of codegen utility functions; I’m sure this isn’t the only place imports will be desired.
```scala
codeParens.spaced(Seq("encoding/json", ???, "net/http").map(enquote))
```